lms/static/certificates/sass/_components.scss

```diff
@@ -58,6 +58,9 @@
     @extend %depth-card;
 }
 
+.sr {
```

Comment:
This is not needed as the Pattern Library utilities already cover this. Please see https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Fbase%2F_reset.scss#L105 and https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Futilities%2F_helpers.scss#L45